test/specs/modules/Progress/Progress-test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {Progress} from 'stardust';
+
+describe('Progress', () => {
+  it('should be able to receive children', () => {
+    render(
+      <Progress>
+        <div className='new-child' />
+      </Progress>
+    ).scryClass('new-child');
```

Comment:
`scry` methods return arrays but do not throw if there are 0 results.  So, this test has no assertion it will always pass.  You can change it to `findClass` which will throw an error if it finds 0, or more than 1.  Or, we've added this `assertText` which does a simple assertion that text is present (text is still a child):

```jsx
render(<Progress>child</Progress>)
  .assertText('child');
```